### PR TITLE
Telemetry: log version in errors

### PR DESCRIPTION
**Pull request by dcposch**
@mathiasvr @Flet

### CODE REVIEW

**Comment by dcposch** [comment]
@feross

File: src/renderer/lib/telemetry.js [hunk 1]
```diff
@@ -118,29 +119,60 @@ function getApproxNumTorrents (state) {
...
-  var message, stack
-  if (err instanceof Error) {
-    message = err.message
-    // Remove the first part of each file path in the stack trace.
-    // - Privacy: remove personal info like C:\Users\<full name>
-    // - Aggregation: this lets us find which stacktraces occur often
-    stack = err.stack.replace(/\(.*app.asar/g, '(...')
+  var message
+  var stack = ''
+  if (e.message) {
```
Comment (mathiasvr) [inline src/renderer/lib/telemetry.js]: I don't think that `e` can be null/undefined now, but maybe we should check that anyway to be sure. We can always remove it if we don't see anything in the telemetry reports, but if it happens the report will never be logged or sent.

**Comment by mathiasvr** [comment]
LGTM

File: src/renderer/lib/telemetry.js (hunk 1 above)
Comment (feross) [inline src/renderer/lib/telemetry.js]: Not a bad idea.

**Pull request merged**